db/column_family.cc

```diff
@@ -492,6 +492,22 @@ ColumnFamilyOptions ColumnFamilyData::GetLatestCFOptions() const {
   return BuildColumnFamilyOptions(initial_cf_options_, mutable_cf_options_);
 }
 
+uint64_t ColumnFamilyData::OldestLogReferenced() {
```

Comment:
Rename it to something like OldestLogToKeep() or something like that?

For non-2PC case, nothing is referenced, but it still returns a log number to keep.